docker-compose.override.yaml

```diff
@@ -4,11 +4,11 @@ version: '3.8'
 # to provide dependencies and expose ports for local testing
 
 services:
-  ffc-template-node:
+  ffc-sfi-api-gateway:
     build:
       target: development
-    image: ffc-template-node-development
-    container_name: ffc-template-node-development
+    image: ffc-sfi-api-gateway-development
+    container_name: ffc-sfi-api-gateway-development
     ports:
       - "3000:3000"
```

Comment:
Might want to map this to a different port if we're going to be running it locally with the other docker containers. I think 3005 is the next available one (ffc-sfi-agreement-viewer is at 3004)